.devcontainer/Dockerfile

```diff
@@ -31,17 +31,18 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive && \
     default-libmysqlclient-dev mariadb-client \
     # LDAP support
     libldap2-dev libsasl2-dev && \
-    apt-get autoclean && apt-get autoremove
+    apt-get autoclean && apt-get autoremove && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Why do we actually remove the package lists? Doesn't that mean if we want to install some apt packages for testing, e.g. the cups headers or the ldap package I have to run apt update before?

Reply:
In most use cases you won't need the lists and running an apt update before installing is always recommended (see recent deployment problems with our own packages with an old access token in the apt lists)

Reply:
Ok. Makes sense